src/foam/nanos/bench/BenchmarkRunner.js

```diff
@@ -182,7 +194,19 @@ foam.CLASS({
       name: 'runScript',
       javaCode: `
       long startTime = System.currentTimeMillis();
+      Logger logger = (Logger) x.get("logger");
+      LogLevelFilterLogger loggerFilter = null;
+      while ( true ) {
+        if ( logger instanceof LogLevelFilterLogger ) {
+          loggerFilter = (LogLevelFilterLogger) logger;
+          break;
+        }
+        logger = ((ProxyLogger) logger).getDelegate();
+      }
+      boolean savedDebugLoggingEnabled = loggerFilter.getLogDebug();
```

Comment:
info log level is not preserved here; function always restores it to true. we can assume that log level info is always turned on but to be safe we should preserve and restore the setting as it is